kubernetes/schema_token_request.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package kubernetes
+
+import (
+	"errors"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	apiv1 "k8s.io/api/authentication/v1"
+)
+
+func tokenRequestSpecFields() map[string]*schema.Schema {
+	s := map[string]*schema.Schema{
+		"audiences": {
```

Comment:
These attributes need to be marked computed as well as you might specify only `audiences` and then `expiration_seconds` will get computed for you.